help/en/html/tools/Routes.shtml

```diff
@@ -205,19 +205,20 @@
             the same as one of the System Names shown in the Route Table.</li>
           </ul>
           The <strong>Add Route</strong> and <strong>Edit Route</strong> buttons in the Add/Edit
-          Route pane will change to <strong>Update Route</strong> and <strong>Cancel</strong>.
+          Route pane will change to <strong>Update</strong> and <strong>Cancel</strong>.
```

Comment:
This &lt;li&gt; section needs to be reduced to clicking the Edit button in the Route table.

Reply:
Thanks.  Could you check the newly-pushed version to see if it's right now?  Thanks again.